.changeset/polite-colts-joke.md

```diff
@@ -0,0 +1,6 @@
+---
+'@commercetools-uikit/select-input': minor
+'@commercetools-uikit/select-utils': minor
+---
+
+provide min and max width props to the select input component. These props are a range of values from the set of horizontal constraints that control the min and max width of the select dropdown menu.
```

Comment:
I'm sorry but I still think we should name the new props here.

What about something like this?
```suggestion
We're introducing two new props (`minMenuWidth` and `maxMenuWidth`) with allow to control the minimum and maximum widths of the select dropdown menu.
```